Author: Michael Biebl <biebl@debian.org>
Description: Avoid a circular dependency between -daemon and libcryptui by
 statically linking seahorse-daemon to libcryptui.


--- a/daemon/Makefile.am
+++ b/daemon/Makefile.am
@@ -74,7 +74,7 @@
 	$(BUILT_SOURCES)
 
 seahorse_daemon_LDADD = \
-	$(top_builddir)/libcryptui/libcryptui.la \
+	$(top_builddir)/libcryptui/libcryptui-private.la \
 	$(top_builddir)/libegg/libeggdatetime.la \
 	$(top_builddir)/libegg/libeggsmclient.la \
 	$(SEAHORSE_LIBS)
--- a/libcryptui/Makefile.am
+++ b/libcryptui/Makefile.am
@@ -34,6 +34,7 @@
 # 
 
 lib_LTLIBRARIES = libcryptui.la
+noinst_LTLIBRARIES = libcryptui-private.la
 
 libcryptui_src = \
     $(srcdir)/cryptui.c \
@@ -55,6 +56,9 @@
     
 libcryptui_la_LIBADD = $(LIBCRYPTUI_LIBS) 
 
+libcryptui_private_la_SOURCES = $(libcryptui_la_SOURCES)
+libcryptui_private_la_LIBADD = $(libcryptui_la_LIBADD)
+
 # ------------------------------------------------------------------
 # PKGCONFIG
 # 
